go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible // TODO: Switch this to bigcache
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.3
-	github.com/streamdal/snitch-protos v0.0.101
+	github.com/streamdal/snitch-protos v0.0.106-0.20231010211049-4e65e47b0eb9
```

Comment:
Is this referencing the right proto version? Is this public/available?

Reply:
It's currently referencing a branch since there are multiple proto changes waiting to go out